src/SelfService/Infrastructure/Api/JsonSchema/SelfServiceJsonSchemaController.cs

```diff
@@ -59,10 +59,9 @@ public async Task<IActionResult> GetSchema(string id, [FromQuery] int schemaVers
 
     [HttpPost("{id:required}")]
     [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest, "application/problem+json")]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status401Unauthorized, "application/problem+json")]
```

Comment:
Is this ever used?
Should it be?

Reply:
![image](https://github.com/dfds/selfservice-api/assets/5738476/e7ad84c5-04e8-4f92-ad2c-0d4a341c4446)
First check can return BadRequest